src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+
```

Comment:
[Please don't add redundant empty lines to your code.](https://github.com/mate-academy/jv-oop/blob/master/checklist.md#please-dont-add-redundant-empty-lines-to-your-code)


check other places

Reply:
@Elena-Bruyako unfortunately, I couldn't remove the spaces between the package name and the class name because it's a code style (mvn clean package shows me an error message when I do that). In other places I removed the extra spaces.